### PR TITLE
Enlist Cosmos DB scaler on Artifact Hub

### LINKED ISSUE #7: List Cosmos DB external scaler on Artifact Hub
Artifact Hub now supports listing KEDA scalers and it would be nice to list Cosmos DB external scaler on Artifact Hub as well.

We already do this for our HTTP add-on scaler - https://artifacthub.io/packages/keda-scaler/keda-official-external-scalers/keda-add-ons-http.

This should be contributed to https://github.com/kedacore/external-scalers/tree/master/artifacthub and updated for every release.

### PR DESCRIPTION
**Pull request by JatinSanghvi**
Added package for 'KEDA external scaler for Azure Cosmos DB' in 'KEDA Official External Scalers' repository on Artifact Hub.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes https://github.com/kedacore/keda-external-scaler-azure-cosmos-db/issues/7

### CODE REVIEW

File: artifacthub/azure-cosmos-db/0.0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# KEDA External Scaler for Azure Cosmos DB
+
+Scale applications based on changes to [Azure Cosmos DB](https://azure.microsoft.com/services/cosmos-db/) container data.
+
+## Getting Started
+
+Please refer to the [official Readme on GitHub](https://github.com/kedacore/keda-external-scaler-azure-cosmos-db#readme) for setup instructions to be able to leverage KEDA and KEDA external scaler for Azure Cosmos DB for scaling your Kubernetes application.
```
Comment (tomkerkhove) [inline artifacthub/azure-cosmos-db/0.0.1/README.md]: We should provide a more decent README as this will appear on Artifact Hub

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
We'll need a bit more docs

**Comment by tomkerkhove** [comment]
Can you also provide the `install` section as per https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml#L31 please?

This should help them install the external scaler.

Example: https://artifacthub.io/packages/keptn/keptn-integrations/dynatrace-service?modal=install ([source](https://github.com/keptn-sandbox/artifacthub/blob/main/dynatrace/0.17.0/artifacthub-pkg.yml))

**Comment by JatinSanghvi** [comment]
Marked PR as draft. I will get the release created and Helm chart ready for the external scaler before working on this one.

**Comment by tomkerkhove** [comment]
Feel free to let me know if you need help here @JatinSanghvi (but take your time)

File: artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+name: keda-external-scaler-azure-cosmos-db
+displayName: KEDA External Scaler for Azure Cosmos DB
+createdAt: 2021-11-11T00:00:00Z
+description: Scale applications based on changes to Azure Cosmos DB container data.
+license: Apache-2.0
+homeURL: https://github.com/kedacore/keda-external-scaler-azure-cosmos-db
+prerelease: true
+keywords:
+  - keda
+  - azure-cosmos-db
```
Comment (tomkerkhove) [inline artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml:14]: Let's throw in a new one
```suggestion
  - azure
  - azure-cosmos-db
```

File: artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - autoscaling
+links:
+  - name: GitHub
+    url: https://github.com/kedacore/keda-external-scaler-azure-cosmos-db
```
Comment (tomkerkhove) [inline artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml]: Let's add our Helm registry to be complete
```suggestion
links:
  - name: Helm Registry
    url: https://github.com/kedacore/charts
  - name: GitHub
    url: https://github.com/kedacore/keda-external-scaler-azure-cosmos-db
```

File: artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      helm install keda kedacore/keda --namespace keda --create-namespace
+      ```
+
+  1. Install 'Cosmos DB scaler' Helm chart.
+
+      ```shell
+      helm install cosmosdb-scaler kedacore/keda-external-scaler-azure-cosmos-db --namespace keda --create-namespace
+      ```
+
+  Please refer to the [setup instructions](https://github.com/kedacore/keda-external-scaler-azure-cosmos-db#readme) for more details.
```
Comment (tomkerkhove) [inline artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml]: Let's align this with our docs from the other PRs when they are merged

File: artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.1.0
+name: keda-external-scaler-azure-cosmos-db
+displayName: KEDA External Scaler for Azure Cosmos DB
+createdAt: 2021-11-11T00:00:00Z
+description: Scale applications based on changes to Azure Cosmos DB container data.
```
Comment (tomkerkhove) [inline artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml]: Let's align with Helm chart description

File: artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.1.0
+name: keda-external-scaler-azure-cosmos-db
```
Comment (tomkerkhove) [inline artifacthub/azure-cosmos-db/0.1.0/artifacthub-pkg.yml]: Let's align with Helm chart name

**Review by tomkerkhove — COMMENTED** [review]
LGTM, but let's wait for https://github.com/kedacore/keda-external-scaler-azure-cosmos-db/pull/20 & https://github.com/kedacore/charts/pull/203

**Review by tomkerkhove — APPROVED** [review]
LGTM, thanks a ton!

**Pull request merged**